scripts/gis-material-update/process/modules/cycling_infra.py

```diff
@@ -48,26 +49,36 @@ def persist_to_database(self):
         )
 
         # persist polygons to database
+        #self._process_result_polygons.to_postgis(
+        #    "cycle_infra_polys",
+        #    engine,
+        #    "public",
+        #    if_exists="replace",
+        #    index=True,
+        #    index_label="fid",
+        #)
+
+        # persist results to temp table
         self._process_result_polygons.to_postgis(
-            "cycle_infra_polys",
+            self._cfg.tormays_table_temp(self._module),
```

Comment:
Extra tables are created to the database. E.g. in this file, there's also `cycle_infra` and in hsl.py, there's `buses_lines`. There are similar hardcoded debug tables in almost every process file. If the database configuration points to the main haitaton database, these debug tables will be written there as well. And we clearly don't want them there.

Reply:
Removed all unnecessary table creations and added possibility to add original data tables if needed.